<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MENÜ EKRANI</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_cikis">
      <property name="text">
       <string>Çıkış</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>500</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_faturaolustur">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>E-FATURA OLUŞTUR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_faturagoruntule">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>FATURALARI GÖRÜNTÜLE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
